developers/index.html

```diff
@@ -27,8 +27,8 @@ <h2>SQL References</h2>
 <ul>
     <li><a href="https://dev.mysql.com/doc/">MySQL</a></li>
     <li><a href="http://www.postgresql.org/docs/">PostgreSQL</a></li>
-    <li><a href="http://www.oracle.com/technology/documentation/database.html">Oracle</a></li>
```

Comment:
I think the entry point to Oracle's reference/documentation is here:
https://www.oracle.com/database/technologies/appdev/sql.html